xbmc/utils/BitstreamConverter.cpp

```diff
@@ -726,6 +727,18 @@ bool CBitstreamConverter::BitstreamConvertInitAVC(void *in_extradata, int in_ext
       if (unit_nb)
         pps_seen = 1;
     }
+
+    if (!unit_nb && !mvc_done++)
+    {
+      if (in_extrasize - total_size > 14 && memcmp(extradata + 8, "mvcC", 4) == 0)
```

Comment:
Generally magic numbers are frowned upon. Please use constexpr int's to help with human readable values. In reality it would just be nice if we could define a structure for the extradata so we don't need to do all this magic number jumping around but I'll leave that as a future wish.

```suggestion
      constexpr int MINIMUM_SIZE{14};
      constexpr int HEADER_OFFSET{8};

      if (in_extrasize - total_size > MINIMUM_SIZE && memcmp(extradata + HEADER_OFFSET, "mvcC", strlen("mvcC")) == 0)
```